content/articles/image-compression-using-PCA/index.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /image-compression-using-PCA/
+title: Image Compression Using PCA
+description: In this article we discuss the dimensionality reduction algorithm, Principal Component Analysis and use the same to compress images.
+author: deewakar-chakraborty
+date: 2021-06-10T00:00:00-18:00
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  -url: /engineering-education/image-compression-using-PCA/hero.png
+   alt: Image compression using PCA example image
+---
+
+Principal Component Analysis, PCA, is a dimensionality reduction method used to reduce the dimensionality of a dataset by transforming the data to a new basis where the dimensions are non-redundant(low covariance) and have high variance. This tutorial aims to make the reader understand the concept of PCA(Principal Component Analysis) mathematically by providing them with one of the use cases of PCA, image compression.
```

Comment:
Explain the basic terminologies used in Linear Algebra:

1. row/column vectors
2. basis 
3. covariance
and other terms used.